tests/integration/dogshell/test_dogshell.py

```diff
@@ -9,6 +9,7 @@
 import time
 import tempfile
 import unittest
+import urllib2
```

Comment:
Let's use `requests` instead. Easier, shorter and compatible Python 2.6—3.4  (not the case for urllib2) !